packages/interaction/src/interactiveTarget.ts

```diff
@@ -126,9 +126,10 @@ export const interactiveTarget = {
      * @memberof PIXI.DisplayObject#
      * @private
      */
-    get trackedPointers()
+    get trackedPointers(): Map<number, InteractionTrackingData>
     {
-        if (this._trackedPointers === undefined) this._trackedPointers = {};
+        if (this._trackedPointers === undefined)
+        { this._trackedPointers = new Map<number, InteractionTrackingData>(); }
```

Comment:
That's behavious change. Map !== Object because people have to use `map.get(key)` instead of `map[key]` . 

`get` and `set` aren't used in InteractiveManager

https://github.com/pixijs/pixi.js/blob/cd8dffb5cc74d5deb4616ef671a9d140d1897bce/packages/interaction/src/InteractionManager.js#L1204

Its strange that TS allowed you to build this

Reply:
Also please don’t use Map, it’s not supported in IE 10

Reply:
@KennethGomez would you mind taking another look at this ☝️